From 9a85a572e87a35e757747775d34949ecdbe6ec4f Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Tue, 10 Jul 2018 14:20:57 +0200
Subject: [PATCH 19/19] Remove max aspect ratio.

Make sure the app is displayed in full-screen on devices with
18:9 and 19:9 screen by removing the default max aspect ratio set for
apps that target older API levels than O.

Change-Id: I715d254e98af261ce6b881bb0c2b79eb32b4bd9c
---
 AndroidManifest.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index deb19a09a..370250d27 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -87,6 +87,9 @@
         android:supportsRtl="true"
         android:usesCleartextTraffic="true" >
 
+        <!-- We can deal with all kinds of aspect ratios -->
+        <meta-data android:name="android.max_aspect" android:value="0.0" />
+
         <uses-library android:name="org.apache.http.legacy" android:required="false" />
         <activity
             android:name="com.android.email.activity.ComposeActivityEmail"
-- 
2.17.1

